libicyque.c

```diff
@@ -1225,6 +1350,75 @@ icq_fetch_events_cb(IcyQueAccount *ia, JsonObject *obj, gpointer user_data)
 	icq_fetch_url_with_method(ia, "GET", fetchBaseURL, NULL, icq_fetch_events_cb, NULL);
 }
 
+static void
+icq_robusto_add_client_cb(IcyQueAccount *ia, JsonObject *obj, gpointer user_data)
+{
+	JsonObject *status = json_object_get_object_member(obj, "status");
+	if(json_object_get_int_member(status, "code") != 20000) {

```

Comment:
does `status` need a null check for safety?

Reply:
In all requests I did (malformed or not), the fields `status` and `results` have always been available. Should I add it anyway?

Reply:
Ah well... it doesn't hurt!